ubuntu/packages

```diff
@@ -46,3 +46,4 @@ python3-dev
 python3-numpy
 srm-ifce-dev
 unixodbc-dev
+wget
```

Comment:
we want `packages` to also serve as documentation of ROOT dependencies on ubuntu, so it would be better to just append `wget` to the `apt install` command run by the Dockerfile